src/views/objects/object/index.js

```diff
@@ -0,0 +1,5 @@
+import ObjectRoute from './ObjectRoute'
+import ObjectHeader from './ObjectHeader/ObjectHeader'
+import ObjectStats from './ObjectStats/ObjectStats'
+
+export { ObjectRoute, ObjectStats, ObjectHeader }
```

Comment:
Here we input all the Object related components, then export them back out. This allows us to import components with the Syntax `import { ObjectStats } from ./views/objects/object/`